_slides/git.html

```diff
@@ -111,95 +127,124 @@ <h3>Autres VCS</h3>
 			<li class="fragment">Mercurial</li>
 			<li class="fragment">Bazaar</li>
 		</ul>
+
+		<aside class="notes">
+			<p>That said, Git is neither (nor the last) VCS, “only” the most popular currently.</p>
+			<p>Here are a few others.</p>
+		</aside>
 	</section>
 
 	<section>
-		<h3>Ce que Git n'est pas</h3>
+		<h3>What Git is not</h3>
 
 		<p class="fragment">Git ≠ GitHub, GitLab, Gitea, etc.</p>
-		<p class="fragment">Git = outil logiciel (en ligne de commande)</p>
-		<p class="fragment">GitHub et GitLab = services d'hébergement de dépôts Git</p>
-		<p class="fragment">Linux ≠ OVH, par exemple</p>
+		<p class="fragment">Git = software tool (command line)</p>
+		<p class="fragment">GitHub et GitLab = Git repository hosting services</p>
+		<p class="fragment">Linux ≠ OVH, for example</p>
+
+		<aside class="notes">
+			<p>Since Git is often used in tandem with GitHub, GitLab, etc., many confuse the two!</p>
+			<p>GitHub, GitLab, etc. provide functionalities complementing Git's, but still, saying “I'm using GitHub on my computer” makes no sense.</p>
+			<p>The difference between Git and Git{Hub,Lab,tea} is equivalent to the difference between Linuw and OVH, for example.</p>
+		</aside>
 	</section>
 </section>
 
 <section>
-	<h2>Pourquoi utiliser Git&nbsp;?</h2>
-	<p class="fragment">Bénéfices sur plusieurs niveaux&nbsp;:</p>
+	<h2>Why use Git?</h2>
+	<p class="fragment">Benefits on several levels:</p>
 	<ol>
-		<li class="fragment">Système de backup</li>
-		<li class="fragment">Travail par "branches"</li>
-		<li class="fragment">Travail collaboratif</li>
+		<li class="fragment">Backup system</li>
+		<li class="fragment">Work in “branches”</li>
+		<li class="fragment">Collaborative work</li>
 	</ol>
-	<p class="fragment"><strong>Ça vaut le coup&nbsp;!</strong></p>
+	<p class="fragment"><strong>It's worth the trouble!</strong></p>
+
+	<aside class="notes">
+		<p>Especially for beginners, the point of using Git may not be obvious, especially for small solo projects.</p>
+		<p>Yet, Git's features are (in my opinion) simple to use, and useful to everyone!</p>
```

Comment:
```suggestion
		<p>Yet, Git's features are (IMO) simple to use, and useful to everyone!</p>
```

Reply:
I want to try and make this accessible to non-native speakers, so I'd rather avoid abbreviations. Plus I don't think it flows well here :stuck_out_tongue: